packages/webui/src/client/ui/PreviewPopUp/Previews/IFramePreview.tsx

```diff
@@ -16,7 +16,8 @@ export function IFramePreview({ content }: IFramePreviewProps): React.ReactEleme
 
 	const onLoadListener = useCallback(() => {
 		if (content.postMessage) {
-			iFrameElement.current?.contentWindow?.postMessage(content.postMessage)
+			const url = new URL(content.href)
+			iFrameElement.current?.contentWindow?.postMessage(content.postMessage, `${url.protocol}//${url.host}`)
```

Comment:
I think a more fool-proof (in terms of port specifiers, etc.) would be to use the [`origin` property](https://developer.mozilla.org/en-US/docs/Web/API/URL/origin). I see that the previous implementation used `*`. I suppose using `url.origin` is safer.

```suggestion
			iFrameElement.current?.contentWindow?.postMessage(content.postMessage, url.origin)
```

Reply:
Good spot, I copied that pattern from elsewhere in core, but this makes way more sense.